nima_core/diagnostics/deps_checks.py

```
@@ -0,0 +1,233 @@
+        return CheckResult(
+            name="deps.openclaw_cli",
+            category=_CATEGORY,
+            status=Status.ERROR,
+            summary="`openclaw` not found on PATH",
+            fix_hint=(
+                "Install OpenClaw — see openclaw.com / GitHub install instructions. "
+                "On macOS: `brew install openclaw` or follow the npm install path."
+            ),
+        )
+    try:
+        version = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)